content/sensu-go/6.2/operations/deploy-sensu/secure-sensu.md

```diff
@@ -210,22 +213,20 @@ The `Subject:` field indicates the certificate's CN is `client`, so to bind the
 
 To enable agent mTLS authentication:
 
-1. Create and distribute new certificates and keys according to the [Generate certificates][12] guide.
-
-2. With the TLS certificate and key are in place, [update the agent configuration using `cert-file` and `key-file` security configuration flags][7].
```

Comment:
I removed this because I think this is what we're telling them to do in the original step 4 already, right?

Reply:
:+1: